spec/mongoid_spec.rb

```diff
@@ -117,7 +117,7 @@ def extension_white_list
         @doc.save!
         @doc.reload
 
-        expect(JSON.parse({:data => @doc.image}.to_json)).to eq("data"=>{"image"=>{"url"=>"/uploads/test.jpeg"}})
+        expect(JSON.parse({:data => @doc.image}.to_json)).to eq("data"=>{"url"=>"/uploads/test.jpeg"})
```

Comment:
Serialization to JSON [have changed](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/uploader/serialization.rb#L14). It is no longer wrapped within `mounted_as`.